lib/db/files_db.dart

```diff
@@ -427,16 +457,26 @@ class FilesDB {
     int creationTime,
   ) async {
     final db = await instance.database;
-    final rows = await db.query(
-      table,
-      where: '''$columnTitle=? AND $columnDeviceFolder=? AND 
+    final rows = await (deviceFolder != null
```

Comment:
Nitpicking, this line could read better. What if we extract `db.query(...)` into a variable, and do a 
```dart
final rows = await query;
``` 
instead?